src/plugins/discourse/mirror.js

```diff
@@ -198,7 +198,10 @@ export class Mirror {
       let offset = 0;
       let upToDate = false;
       while (!upToDate) {
-        const likeActions = await this._fetcher.likesByUser(user, offset);
+        const likeActions = await this._fetcher.likesByUser(
+          user.username,
+          offset
+        );
```

Comment:
nit: revert this file. (surprised, usually prettier is pretty good at not leaving spurious formatting changes)